README.md

```diff
@@ -37,12 +42,25 @@ The action can be used as a step in a workflow
 Here is an example that syncs from the local path `temp/blog` to the blog section of the bucket.
 
 ```yml
-- name: Sync to the blog
-  uses: deephaven/salmon-sync@v1
-  with:
-    source: temp/blog
-    destination: deephaven/deephaven.io/blog
-    production: true # false for pr previews
-    temporary: false # true will delete non-production files after 14 days
-    aws-role: ${{ vars.DOCS_AWS_ROLE }}
+jobs:
+  upload-to-salmon:
+    runs-on: ubuntu-24.04
+    permissions:
+      id-token: write # Needed to authenticate with AWS
+      contents: read # If you want to checkout the repo
+    steps:
+      - name: Checkout the repo
+        uses: actions/checkout@v4
+
+      - name: Build docs
+        run: echo "Build docs here"
```

Comment:
Wonder if we should throw an error or something instead indicating they need to fill in the build step.
Or maybe we have two examples, one showing with a build step and one showing without. I think having just a "Build docs" step that will be successful and just outputs "Build docs here" may be a little misleading.